<ui version="4.0" >
 <class>Form</class>
 <widget class="QWidget" name="Form" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>518</width>
    <height>284</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize" >
   <size>
    <width>518</width>
    <height>284</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>496</width>
    <height>262</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="spacing" >
    <number>0</number>
   </property>
   <property name="leftMargin" >
    <number>0</number>
   </property>
   <property name="topMargin" >
    <number>0</number>
   </property>
   <property name="rightMargin" >
    <number>0</number>
   </property>
   <property name="bottomMargin" >
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frameOuterCosmetic" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize" >
      <size>
       <width>16</width>
       <height>284</height>
      </size>
     </property>
     <property name="maximumSize" >
      <size>
       <width>16777215</width>
       <height>284</height>
      </size>
     </property>
     <property name="palette" >
      <palette>
       <active>
        <colorrole role="WindowText" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>127</red>
           <green>157</green>
           <blue>185</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="WindowText" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>127</red>
           <green>157</green>
           <blue>185</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="WindowText" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>127</red>
           <green>127</green>
           <blue>127</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="autoFillBackground" >
      <bool>true</bool>
     </property>
     <property name="frameShape" >
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QVBoxLayout" >
      <property name="spacing" >
       <number>10</number>
      </property>
      <property name="leftMargin" >
       <number>10</number>
      </property>
      <property name="topMargin" >
       <number>10</number>
      </property>
      <property name="rightMargin" >
       <number>10</number>
      </property>
      <property name="bottomMargin" >
       <number>10</number>
      </property>
      <item>
       <widget class="QFrame" name="frame" >
        <property name="palette" >
         <palette>
          <active>
           <colorrole role="WindowText" >
            <brush brushstyle="SolidPattern" >
             <color alpha="255" >
              <red>208</red>
              <green>208</green>
              <blue>191</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Base" >
            <brush brushstyle="SolidPattern" >
             <color alpha="255" >
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Window" >
            <brush brushstyle="SolidPattern" >
             <color alpha="255" >
              <red>236</red>
              <green>234</green>
              <blue>223</blue>
             </color>
            </brush>
           </colorrole>
          </active>
          <inactive>
           <colorrole role="WindowText" >
            <brush brushstyle="SolidPattern" >
             <color alpha="255" >
              <red>208</red>
              <green>208</green>
              <blue>191</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Base" >
            <brush brushstyle="SolidPattern" >
             <color alpha="255" >
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Window" >
            <brush brushstyle="SolidPattern" >
             <color alpha="255" >
              <red>236</red>
              <green>234</green>
              <blue>223</blue>
             </color>
            </brush>
           </colorrole>
          </inactive>
          <disabled>
           <colorrole role="WindowText" >
            <brush brushstyle="SolidPattern" >
             <color alpha="255" >
              <red>127</red>
              <green>127</green>
              <blue>127</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Base" >
            <brush brushstyle="SolidPattern" >
             <color alpha="255" >
              <red>236</red>
              <green>234</green>
              <blue>223</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Window" >
            <brush brushstyle="SolidPattern" >
             <color alpha="255" >
              <red>236</red>
              <green>234</green>
              <blue>223</blue>
             </color>
            </brush>
           </colorrole>
          </disabled>
         </palette>
        </property>
        <property name="autoFillBackground" >
         <bool>true</bool>
        </property>
        <property name="frameShape" >
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow" >
         <enum>QFrame::Plain</enum>
        </property>
        <layout class="QVBoxLayout" >
         <property name="spacing" >
          <number>10</number>
         </property>
         <property name="leftMargin" >
          <number>10</number>
         </property>
         <property name="topMargin" >
          <number>10</number>
         </property>
         <property name="rightMargin" >
          <number>10</number>
         </property>
         <property name="bottomMargin" >
          <number>10</number>
         </property>
         <item>
          <widget class="QLabel" name="labelMessage" >
           <property name="palette" >
            <palette>
             <active>
              <colorrole role="WindowText" >
               <brush brushstyle="SolidPattern" >
                <color alpha="255" >
                 <red>0</red>
                 <green>0</green>
                 <blue>0</blue>
                </color>
               </brush>
              </colorrole>
             </active>
             <inactive>
              <colorrole role="WindowText" >
               <brush brushstyle="SolidPattern" >
                <color alpha="255" >
                 <red>0</red>
                 <green>0</green>
                 <blue>0</blue>
                </color>
               </brush>
              </colorrole>
             </inactive>
             <disabled>
              <colorrole role="WindowText" >
               <brush brushstyle="SolidPattern" >
                <color alpha="255" >
                 <red>118</red>
                 <green>116</green>
                 <blue>108</blue>
                </color>
               </brush>
              </colorrole>
             </disabled>
            </palette>
           </property>
           <property name="font" >
            <font>
             <family>Arial</family>
             <pointsize>9</pointsize>
            </font>
           </property>
           <property name="text" >
            <string>Choose a server certificate to validate against:</string>
           </property>
           <property name="wordWrap" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_2" >
           <property name="frameShape" >
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow" >
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" >
            <property name="spacing" >
             <number>14</number>
            </property>
            <property name="leftMargin" >
             <number>18</number>
            </property>
            <property name="topMargin" >
             <number>0</number>
            </property>
            <property name="rightMargin" >
             <number>18</number>
            </property>
            <property name="bottomMargin" >
             <number>0</number>
            </property>
            <item>
             <widget class="QTableWidget" name="tableCertList" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize" >
               <size>
                <width>0</width>
                <height>126</height>
               </size>
              </property>
              <property name="maximumSize" >
               <size>
                <width>16777215</width>
                <height>126</height>
               </size>
              </property>
              <property name="font" >
               <font>
                <family>Arial</family>
                <pointsize>9</pointsize>
               </font>
              </property>
              <property name="frameShape" >
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="verticalScrollBarPolicy" >
               <enum>Qt::ScrollBarAlwaysOn</enum>
              </property>
              <property name="horizontalScrollBarPolicy" >
               <enum>Qt::ScrollBarAlwaysOff</enum>
              </property>
              <property name="sortingEnabled" >
               <bool>true</bool>
              </property>
              <property name="rowCount" >
               <number>5</number>
              </property>
              <property name="columnCount" >
               <number>2</number>
              </property>
              <row/>
              <row/>
              <row/>
              <row/>
              <row/>
              <column>
               <property name="text" >
                <string/>
               </property>
              </column>
              <column>
               <property name="text" >
                <string>Name</string>
               </property>
              </column>
             </widget>
            </item>
            <item>
             <widget class="QFrame" name="frame_4" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="frameShape" >
               <enum>QFrame::NoFrame</enum>
              </property>
              <property name="frameShadow" >
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QVBoxLayout" >
               <property name="leftMargin" >
                <number>0</number>
               </property>
               <property name="topMargin" >
                <number>0</number>
               </property>
               <property name="rightMargin" >
                <number>0</number>
               </property>
               <property name="bottomMargin" >
                <number>0</number>
               </property>
               <item>
                <widget class="QCheckBox" name="checkBoxVerifyName" >
                 <property name="sizePolicy" >
                  <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="palette" >
                  <palette>
                   <active>
                    <colorrole role="WindowText" >
                     <brush brushstyle="SolidPattern" >
                      <color alpha="255" >
                       <red>0</red>
                       <green>0</green>
                       <blue>0</blue>
                      </color>
                     </brush>
                    </colorrole>
                   </active>
                   <inactive>
                    <colorrole role="WindowText" >
                     <brush brushstyle="SolidPattern" >
                      <color alpha="255" >
                       <red>0</red>
                       <green>0</green>
                       <blue>0</blue>
                      </color>
                     </brush>
                    </colorrole>
                   </inactive>
                   <disabled>
                    <colorrole role="WindowText" >
                     <brush brushstyle="SolidPattern" >
                      <color alpha="255" >
                       <red>127</red>
                       <green>127</green>
                       <blue>127</blue>
                      </color>
                     </brush>
                    </colorrole>
                   </disabled>
                  </palette>
                 </property>
                 <property name="font" >
                  <font>
                   <family>Arial</family>
                   <pointsize>9</pointsize>
                  </font>
                 </property>
                 <property name="text" >
                  <string>Verify Common Name</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QFrame" name="frame_3" >
                 <property name="frameShape" >
                  <enum>QFrame::NoFrame</enum>
                 </property>
                 <property name="frameShadow" >
                  <enum>QFrame::Raised</enum>
                 </property>
                 <layout class="QHBoxLayout" >
                  <property name="spacing" >
                   <number>0</number>
                  </property>
                  <property name="leftMargin" >
                   <number>0</number>
                  </property>
                  <property name="topMargin" >
                   <number>0</number>
                  </property>
                  <property name="rightMargin" >
                   <number>0</number>
                  </property>
                  <property name="bottomMargin" >
                   <number>0</number>
                  </property>
                  <item>
                   <spacer>
                    <property name="orientation" >
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeType" >
                     <enum>QSizePolicy::Fixed</enum>
                    </property>
                    <property name="sizeHint" >
                     <size>
                      <width>18</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                  <item>
                   <widget class="QFrame" name="frame_5" >
                    <property name="sizePolicy" >
                     <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="frameShape" >
                     <enum>QFrame::NoFrame</enum>
                    </property>
                    <property name="frameShadow" >
                     <enum>QFrame::Raised</enum>
                    </property>
                    <layout class="QVBoxLayout" >
                     <property name="spacing" >
                      <number>1</number>
                     </property>
                     <property name="leftMargin" >
                      <number>0</number>
                     </property>
                     <property name="topMargin" >
                      <number>0</number>
                     </property>
                     <property name="rightMargin" >
                      <number>0</number>
                     </property>
                     <property name="bottomMargin" >
                      <number>0</number>
                     </property>
                     <item>
                      <widget class="QLineEdit" name="dataFieldCommonName" >
                       <property name="sizePolicy" >
                        <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
                         <horstretch>0</horstretch>
                         <verstretch>0</verstretch>
                        </sizepolicy>
                       </property>
                       <property name="minimumSize" >
                        <size>
                         <width>260</width>
                         <height>22</height>
                        </size>
                       </property>
                       <property name="font" >
                        <font>
                         <family>Arial</family>
                         <pointsize>9</pointsize>
                        </font>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QLabel" name="labelNameInstructions" >
                       <property name="sizePolicy" >
                        <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
                         <horstretch>0</horstretch>
                         <verstretch>0</verstretch>
                        </sizepolicy>
                       </property>
                       <property name="minimumSize" >
                        <size>
                         <width>0</width>
                         <height>18</height>
                        </size>
                       </property>
                       <property name="palette" >
                        <palette>
                         <active>
                          <colorrole role="WindowText" >
                           <brush brushstyle="SolidPattern" >
                            <color alpha="255" >
                             <red>0</red>
                             <green>0</green>
                             <blue>0</blue>
                            </color>
                           </brush>
                          </colorrole>
                         </active>
                         <inactive>
                          <colorrole role="WindowText" >
                           <brush brushstyle="SolidPattern" >
                            <color alpha="255" >
                             <red>0</red>
                             <green>0</green>
                             <blue>0</blue>
                            </color>
                           </brush>
                          </colorrole>
                         </inactive>
                         <disabled>
                          <colorrole role="WindowText" >
                           <brush brushstyle="SolidPattern" >
                            <color alpha="255" >
                             <red>127</red>
                             <green>127</green>
                             <blue>127</blue>
                            </color>
                           </brush>
                          </colorrole>
                         </disabled>
                        </palette>
                       </property>
                       <property name="font" >
                        <font>
                         <family>Arial</family>
                         <pointsize>8</pointsize>
                        </font>
                       </property>
                       <property name="text" >
                        <string>Use "*" for prefix wildcarding.  For example: *.utah.edu"</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </widget>
                  </item>
                  <item>
                   <spacer>
                    <property name="orientation" >
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" >
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <spacer>
              <property name="orientation" >
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" >
               <size>
                <width>20</width>
                <height>1</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
